Library/Formula/nq.rb

```diff
@@ -0,0 +1,21 @@
+class Nq < Formula
+  desc "Unix command-line queue utility"
+  homepage "https://github.com/chneukirchen/nq"
+  url "https://github.com/chneukirchen/nq/archive/v0.1.tar.gz"
+  sha256 "e0962a5e6a2cbf799bba7a79af22c40d21e5a80605df42c8bba37d3d8efb1793"
+
+  head "https://github.com/chneukirchen/nq.git"
+
+  depends_on :macos => :yosemite
```

Comment:
Perhaps a typo upstream then, 10.3 instead of 10.10.3 or such. No worries. Thanks for the clarification!